src/main/resources/application.properties

```diff
@@ -1 +1,7 @@
 spring.application.name=hng-java-boilerplate
```

Comment:
I hope you won't push prod details like this to GitHub

Reply:
I only pushed the application.properties because of the the task asking for configuration and considering it's still in developement mode, and the strings will be changed, i felt there is no need to use environmental variables or not push to github